components/homme/src/theta-l/imex_mod.F90

```diff
@@ -288,15 +313,21 @@ subroutine compute_stage_value_dirk(nm1,alphadt_nm1,n0,alphadt_n0,np1,dt2,qn0,el
                       end if
                    end do
                    alpha = alpha/2
```

Comment:
Just a question, given this loop does this mean that `alpha` is always 1/2 when it is assigned to `alphas`

Reply:
It's always <= 1/2. The loop finds a distance <= 1, and then that distance is cut in half.